powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationTemplateUpdateRequest.java

```diff
@@ -34,15 +39,37 @@
 @Data
 public class OperationTemplateUpdateRequest {
 
+    @Schema(description = "Operation template identifier")
+    @NotNull(message = "Operation template identifier must not be empty when updating operation template")
```

Comment:
Wrong constraint reference
```suggestion
    @NotNull(message = "Operation template identifier must not be null when updating operation template")
```